internal/pkg/cli/job_delete.go

```diff
@@ -275,6 +297,29 @@ func (o *deleteJobOpts) deleteStack(sess *awssession.Session, env string) error
 	return nil
 }
 
+func (o *deleteJobOpts) deleteTasks(sess *session.Session, env string) error {
+	cluster, err := o.newClusterGetter(sess).Cluster(o.appName, env)
+	if err != nil {
+		return fmt.Errorf("get cluster for env %s: %w", env, err)
+	}
+	ecsClient := o.newECSClient(sess)
+	tdFamilyName := fmt.Sprintf(ecs.FmtWorkloadTaskDefinitionFamily, o.appName, env, o.name)
+	tasks, err := ecsClient.FamilyTasks(cluster, tdFamilyName)
```

Comment:
I wonder if this should be vended within the  new`ecs` pkg now. Maybe a method signature like this:

```go
// Internally calls Cluster(appName, env) and then lists the tasks using the `aws/ecs` pkg.
func (c Client) ListTasks(appName, envName string, filters ...ListTaskFilter) (taskARNs []string, err error)
```
and a sample client that uses this method:
```go
arns, err := c.ListTasks(o.appName, env, ecs.FilterOnActiveTasks())
```
this way the const can also become private